Directory.Build.props

```diff
@@ -29,6 +29,12 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0' or '$(TargetFramework)' == 'net7.0'">
+    <EnableTrimAnalyzer>true</EnableTrimAnalyzer>
+    <PublishTrimmed>true</PublishTrimmed>
```

Comment:
Is this needed for libraries? I guess it's not a problem to have it there.

Reply:
Those are libraries, only the exec should be trimmed.

Reply:
> Is this needed for libraries? I guess it's not a problem to have it there.

This is not needed and will be replaced with `IsTrimmable`.